src/frontend/src/players/pipelines/rtpUnpacker.ts

```diff
@@ -0,0 +1,73 @@
+import { Transform, TransformCallback, TransformOptions } from "stream"
+
+export class RtpPacket {
+  constructor(
+    public payloadType: number,
+    public sequenceNumber: number,
+    public timestamp: number,
+    public ssrc: number,
+    public payload: Buffer
+  ) {}
+}
+
+
+
+/**
+ * This class implements a Transform stream. 
+ * It is meant to be used as a Rtp packet payload unpacker for MPEGTS (MPEG Transport Stream specified by MPEG-2 part 1). 
+ * Currently, it assumes that each chunk represent one UDP packet of OZI Transport Layer that embed one whole RTP packet of OZI Application Layer.
+ */
+export class RtpUnpacker extends Transform {
+  constructor(options: TransformOptions = {}) {
```

Comment:
Not really needed. Automatically the constructor of Transform class will be used